packages/dx-react-demos/src/material-ui/localization/basic.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {
+  FilteringState,
+  LocalFiltering,
+  GroupingState,
+  LocalGrouping,
+  EditingState,
+  PagingState,
+  LocalPaging,
+} from '@devexpress/dx-react-grid';
+import {
+  Grid,
+  TableView,
+  TableHeaderRow,
+  TableFilterRow,
+  TableGroupRow,
+  GroupingPanel,
+  TableEditRow,
+  TableEditColumn,
+  PagingPanel,
+  DragDropContext,
+} from '@devexpress/dx-react-grid-material-ui';
+
+import {
+  generateRows,
+  globalSalesValues,
+} from '../../demo-data/generator';
+
+const tableViewMessages = {
+  noData: 'Keine Daten verfügba',
+};
+const editColumnMessages = {
+  addCommand: 'Neue Zeile',
+  editCommand: 'Bearbeiten',
+  deleteCommand: 'Entfernen',
+  commitCommand: 'Speichern',
+  cancelCommand: 'Abbrechen',
+};
+const groupingPanelMessages = {
+  groupByColumn: 'Ziehen Sie eine Spalte hierhin, um danach zu gruppieren',
+};
+const filterRowMessages = {
+  filterPlaceholder: 'Filter...',
+};
+const pagingPanelMessages = {
+  showAll: 'Alle',
+  rowsPerPage: 'Zeilen pro Seite',
+  info: 'Zeile {from} von {to}({count} Elemente)',
+};
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      columns: [
+        { name: 'region', title: 'Region' },
+        { name: 'sector', title: 'Sektor' },
+        { name: 'channel', title: 'Kanal' },
+        { name: 'customer', title: 'Kunde' },
+      ],
+      rows: generateRows({ columnValues: globalSalesValues, length: 14 }),
+    };
+
+    // eslint-disable-next-line no-alert
+    this.commitChanges = () => alert('Editing isn\'t implemented in this demo');
```

Comment:
No German here? Should be "Datenänderungen sind in dieser Demo nicht implementiert."